packages/docmanager-extension/src/index.ts

```diff
@@ -42,7 +42,8 @@ const opener: JupyterFrontEndPlugin<IDocumentWidgetOpener> = {
           let route = 'edit';
           if (
             (widgetName === 'default' && ext === '.ipynb') ||
-            widgetName === 'Notebook'
+            widgetName === 'Notebook' ||
+            widgetName === 'Jupytext Notebook'
```

Comment:
Or maybe we can check if `widgetName` contains the string `Notebook`?

`jupytext` is a third-party extension and not officially supported, so not sure we should hardcode its special case here.